### PR TITLE
feat(start_nominating): add rebond guide

### LINKED ISSUE #37: [start_nominating] add information about rebonding if accidentally unbonded
some users unbonded their staked funds by accident in the past.
we should add information on rebonding before the unbonding period ends.
this can be done by submitting staking.rebond in https://polkadot.js.org/apps/#/extrinsics

### PR DESCRIPTION
**Pull request by SimonKraus**
fixes #37
Bildschirmfoto 2021-04-18 um 00 11 43

@vgantchev kindly request your proofreading and correction

### CODE REVIEW

File: docs/start_nominating.md [hunk 1]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
   <img src={useBaseUrl('/nominator-guide/nominate-validator-2.png')} />
 </div>  
 
+## 05 Rebond funds {#05-rebond-funds}
+
+If you accidentally unbonded your HDX tokens and don't want to wait the 28 days unbonding period you can rebond the tokens.
```
Comment (vgantchev) [inline docs/start_nominating.md:156]:
```suggestion
If you have accidentally unbonded your HDX tokens, you can rebond them before the waiting period of 28 days has lapsed.
```

File: docs/start_nominating.md [hunk 2]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
+
+To do so, open Polkadot/apps and navigate to:
```
Comment (vgantchev) [inline docs/start_nominating.md:158]:
```suggestion
To do so, open Polkadot/apps and navigate to *Developer* > *Extrinsics*. Alternatively, you can follow this link:

https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-01.snakenet.hydradx.io#/extrinsics
```

File: docs/start_nominating.md [hunk 3]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
   <img src={useBaseUrl('/nominator-guide/nominate-validator-2.png')} />
 </div>  
 
+## 05 Rebond funds {#05-rebond-funds}
+
+If you accidentally unbonded your HDX tokens and don't want to wait the 28 days unbonding period you can rebond the tokens.
+
+To do so, open Polkadot/apps and navigate to:  
+*Developer* > *Extrinsics*
```
Comment (vgantchev) [inline docs/start_nominating.md:159]:
```suggestion

```

File: docs/start_nominating.md [hunk 4]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
+
+Select your account in the _using the selected account_ dropdown.
```
Comment (vgantchev) [inline docs/start_nominating.md:161]:
```suggestion
Select your account in the _using the selected account_ dropdown.  After that, you need to fill out the following information:

* **extrinsics**: staking
* **action**: rebond_value
* **value**: here you need to enter the amount of HDX you want to rebond.
```

File: docs/start_nominating.md [hunk 5]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
 
+## 05 Rebond funds {#05-rebond-funds}
+
+If you accidentally unbonded your HDX tokens and don't want to wait the 28 days unbonding period you can rebond the tokens.
+
+To do so, open Polkadot/apps and navigate to:  
+*Developer* > *Extrinsics*
+
+Select your account in the _using the selected account_ dropdown.  
+At _submit the following extrinsic_ you will select _staking_ in the first dropdown and _rebond(value)_ in the second.
```
Comment (vgantchev) [inline docs/start_nominating.md:162]:
```suggestion
```

File: docs/start_nominating.md [hunk 6]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
+## 05 Rebond funds {#05-rebond-funds}
+
+If you accidentally unbonded your HDX tokens and don't want to wait the 28 days unbonding period you can rebond the tokens.
+
+To do so, open Polkadot/apps and navigate to:  
+*Developer* > *Extrinsics*
+
+Select your account in the _using the selected account_ dropdown.  
+At _submit the following extrinsic_ you will select _staking_ in the first dropdown and _rebond(value)_ in the second.  
+In the _value_ field you will input the amount of HDX tokens you accidentally unbonded.
```
Comment (vgantchev) [inline docs/start_nominating.md:163]:
```suggestion

```

File: docs/start_nominating.md [hunk 7]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
+
+You should have filled the form similar like shown in the following picture:
```
Comment (vgantchev) [inline docs/start_nominating.md:165]:
```suggestion

```

File: docs/start_nominating.md [hunk 8]
```diff
@@ -151,5 +151,23 @@ Adjusting your nominations can be done on the fly, there is no need to stop nomi
...
+
+<div style={{textAlign: 'center'}}>
+  <img src={useBaseUrl('/nominator-guide/rebond.png')} />
+</div>
+
+If everything is set correctly you can submit the extrinsic by clicking the _Submit Transaction_ button and confirm it by hitting the _Sign and Submit_ button in the popup.
```
Comment (vgantchev) [inline docs/start_nominating.md:171]:
```suggestion
If everything is set correctly you can submit the extrinsic by clicking the _Submit Transaction_ button and sign the transaction in the Polkadot.js extension. Once completed, the selected amount of HDX should be rebonded.
```

**Review by vgantchev — COMMENTED** [review]
Nice addition! Here are some suggestions to make it more consistent with other pages (especially the submit extrinsic part). Not sure if all the inline editing really worked out so please double check it 😄

**Comment by SimonKraus** [comment]
applied and double-checked @vgantchev

**Review by vgantchev — APPROVED** [review]

**Pull request merged**